### PR TITLE
Fix #71 by adding a keyword argument to ignore repeated delimiters

### LINKED ISSUE #71: Provide a way to ignore repeated delimiters while parsing
I was trying to load a data file that occasionally has multiple spaces between observations and was surprised to see this for the following contents:
```
a b c d e
1 2  3 4 5
1 2 3  4 5
1  2 3  4 5
```
```
julia> t = CSV.read("Untitled.txt", delim=' ' , header=true)
3×5 DataFrames.DataFrame
│ Row │ a │ b │ c     │ d     │ e     │
├─────┼───┼───┼───────┼───────┼───────┤
│ 1   │ 1 │ 2 │ #NULL │ 3     │ 4     │
│ 2   │ 5 │ 1 │ 2     │ 3     │ #NULL │
│ 3   │ 4 │ 5 │ 1     │ #NULL │ 2     │
```
Any suggestions on how to resolve this?

### PR DESCRIPTION
**Pull request by quinnj**

### CODE REVIEW

**Comment by quinnj** [comment]
Waiting on a new Parsers.jl release w/ the necessary functionality.

File: src/CSV.jl [hunk 1]
```diff
@@ -233,6 +234,7 @@ function File(source::Union{String, IO},
     missingstrings,
     missingstring,
     delim::Union{Char, String},
+    ignore_repeated_delimiters::Bool,
```
Comment (nalimilan) [inline src/CSV.jl]: Can we use a shorter name, ideally without underscores? I cry everytime I write `rows_for_type_detect`...

Comment (quinnj) [reply src/CSV.jl]: Ugly names for ugly features; that's what I say....

Comment (nalimilan) [reply src/CSV.jl]: https://github.com/JuliaData/CSV.jl/pull/266

**Pull request merged**